sources/packages/backend/apps/api/src/route-controllers/supporting-user/models/supporting-user.dto.ts

```diff
@@ -1,17 +1,30 @@
-import { IsNotEmpty, IsNotEmptyObject, IsOptional } from "class-validator";
-import { ContactInfo, SupportingUserType } from "@sims/sims-db";
+import {
+  IsNotEmpty,
+  IsNotEmptyObject,
+  IsOptional,
+  Length,
+  MaxLength,
+} from "class-validator";
+import {
+  ContactInfo,
+  SupportingUserType,
+  APPLICATION_NUMBER_LENGTH,
+} from "@sims/sims-db";
+
+const STUDENT_LAST_NAME_MAX_LENGTH = 100;
```

Comment:
No need to change right now, but when the MAX constraints represent the DB max value they can be declared at the entity model layer to be reused everywhere.